src/cplus_plugin/gui/priority_group_dialog.py

```diff
@@ -73,7 +73,7 @@ def update_ok_buttons(self):
     def initialize_ui(self):
         """Populate UI inputs when loading the dialog"""
 
-        self.btn_help.setIcon(FileUtils.get_icon("mActionHelpContents.svg"))
+        self.btn_help.setIcon(FileUtils.get_icon("mActionHelpContents_green.svg"))
```

Comment:
Is this file already added?

Reply:
@zamuzakki , yes as its also being used in other UI components.